lib/modules/userTagger.js

```diff
@@ -365,6 +416,10 @@ function populateDialog(tag: Tag, card) {
 				<label class="fieldPair-label" for="userTaggerVotesDown" title="Downvotes you have given this redditor">Downvotes</label>
 				<input type="number" style="width: 50px;" id="userTaggerVotesDown" value="${tag.votesDown}">
 			</div>
+			<div class="fieldPair">
```

Comment:
Add `${!presetTags.length && 'hidden'}` in order to hide when there's no presets.

Reply:
The `fieldPair` won't immediately show when a user clicks 'save as preset' without refreshing the dialog. I was thinking of removing `class="hidden"` with classList.remove() from `elements.presetTag.parentElement` when saveAs is clicked but is there an alternate/simpler way you were thinking?

Reply:
If this has the `hidden` attribute by default, `buildPresetTagElement` can include the line `elements.presetTag.parentElement.hidden = false` to make it visible.

Reply:
`display:flex` in fieldPair is overriding `display:none` in the hidden attribute which is why I was thinking to set up a `hidden` class. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden (see note)

Reply:
Oh, right, that's problematic. Making `display: flex` not apply when the `hidden` attribute is present might be the easiest solution. https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/3b04202a7672a431df4adb5097183a9488ffc564/lib/css/modules/_hover.scss#L71 can be changed to `.fieldPair:not([hidden])`.